cli/src/main.rs

```diff
@@ -65,16 +65,21 @@ enum TapSignerCommand {
         #[clap(short, long, value_delimiter = ',', num_args = 1..)]
         path: Vec<u32>,
     },
+    /// Get a an encrypted backup of the card's private key
+    Backup,
+    /// Change the PIN (CVC) used for card authentication to a new user provided one
+    Change { new_cvc: String },
```

Comment:
Small thing:  It may be more explicit to say add to the work `Change`.  I say because my first thought with Change is the sats you get back in a transaction.

Note: I made the suggestion in the parts of code that would change, so if the suggestion is rejected, you'll know why the other suggestions are there.

```suggestion
    ChangeCVC { new_cvc: String },
```